src/pg/brc20/brc20-pg-store.ts

```diff
@@ -315,26 +327,25 @@ export class Brc20PgStore {
         ON CONFLICT (LOWER(ticker)) DO NOTHING
         RETURNING id
       `;
-      if (insertion.count > 0) {
-        // Add to history
-        const event: DbBrc20EventInsert = {
-          inscription_id: args.inscription_id,
-          brc20_deploy_id: insertion[0].id,
-          deploy_id: insertion[0].id,
-          mint_id: null,
-          transfer_id: null,
-        };
-        await sql`
-          INSERT INTO brc20_events ${sql(event)}
-        `;
-        logger.info(
-          `PgStore [BRC-20] inserted deploy for ${args.deploy.tick} at block ${args.location.block_height}`
-        );
-      } else {
-        logger.debug(
+      if (insertion.count <= 0) {
+        return logger.debug(
           `PgStore [BRC-20] ignoring duplicate deploy for ${args.deploy.tick} at block ${args.location.block_height}`
         );
       }
+      // Add to history
+      const event: DbBrc20EventInsert = {
+        inscription_id: args.inscription_id,
+        brc20_deploy_id: insertion[0].id,
+        deploy_id: insertion[0].id,
+        mint_id: null,
+        transfer_id: null,
+      };
+      await sql`
+        INSERT INTO brc20_events ${sql(event)}
+      `;
+      logger.info(
+        `PgStore [BRC-20] inserted deploy for ${args.deploy.tick} at block ${args.location.block_height}`
+      );
```

Comment:
We should remove this (and also the events table). It is not used anymore (unless you'd want to use it for something? In that case we'll need to replicate this to all other events perhaps in another PR)

Reply:
I'd like to show an activity log (event log?) per ticker kn the explorer. But this be doable by just merging the different events. A separate table would probably make things easier for pagination etc.

Reply:
(so far this was only a refactor reducing some nesting, no changes in logic, I'll remove again to keep things easier)